bindings/node.js/src/kzg.cxx

```diff
@@ -161,6 +162,28 @@ inline Bytes48 *get_bytes48(
     );
 }
 
+Napi::Value GetFieldElementsPerBlob(const Napi::CallbackInfo &info) {
+    Napi::Env env = info.Env();
+    KZGSettings *kzg_settings = get_kzg_settings(env, info);
+    if (kzg_settings == nullptr) {
+        return env.Null();
+    }
+    return Napi::Number::New(
+        env, static_cast<double>(kzg_settings->field_elements_per_blob)
+    );
+}
+
+Napi::Value GetBytesPerBlob(const Napi::CallbackInfo &info) {
+    Napi::Env env = info.Env();
+    KZGSettings *kzg_settings = get_kzg_settings(env, info);
+    if (kzg_settings == nullptr) {
+        return env.Null();
+    }
+    return Napi::Number::New(
+        env, static_cast<double>(kzg_settings->bytes_per_blob)
```

Comment:
I didn't notice where `bytes_per_blob` and `field_elements_per_blob` is getting added to the `KZGSettings`.  Are you sure this doesn't throw/segfault?